http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=370825

/etc/gssapi_mech.conf by default refers to /usr/lib/libgssapi_krb5.so,
but this file doesn't exist. Changing it to /usr/lib/libgssapi_krb5.so.2
improves things.
David Pashley <david@davidpashley.com>

2011-03-24: amended to use a relative path for the mech, for compatibility
with multiarch paths.
Steve Langasek <steve.langasek@linaro.org>

--- a/doc/gssapi_mech.conf	2005-11-23 02:16:31.000000000 +1100
+++ b/doc/gssapi_mech.conf	2009-06-06 18:26:47.000000000 +1000
@@ -15,7 +15,7 @@
 # library                               initialization function
 # ================================	==========================
 # The MIT K5 gssapi library, use special function for initialization.
-/usr/lib/libgssapi_krb5.so		mechglue_internal_krb5_init
+libgssapi_krb5.so.2			mechglue_internal_krb5_init
 #
 # The SPKM3 gssapi library function.  Use the function spkm3_gss_initialize.
 # /usr/local/gss_mechs/spkm/spkm3/libgssapi_spkm3.so    spkm3_gss_initialize
